packages/mobile/views/onboarding/views/app-setup/views/WelcomeView.svelte

```diff
@@ -3,7 +3,7 @@
     import { localize } from '@core/i18n'
```

Comment:
I think that we could change these imports accordingly to the code conventions
https://iotaledger.github.io/firefly/guides/coding-conventions/imports/#order